File: chingoon.rb
#!/usr/bin/env ruby

if ARGV[0] !~ /^\d*$/ then
  raise SyntaxError, "Invalid Chins Argument"
end

chins = ARGV[0].to_i

puts <<DONGS
 __________
(--[ .]-[ .]
DONGS


case chins
when 0
  puts '(_______O__)'
else
  puts '(       O  )'
  (chins - 1).times do
    puts '(          )'
  end
  puts '(__________)'
end

File: chingoon.py
#!/usr/bin/env python

import sys
try:
    chins = long(sys.argv[1])
except IndexError:
    raise IndexError, 'needachin'
except ValueError:
    raise ValueError, 'needintchins'

if chins < 0:
    raise ValueError, 'negachin'


print " __________"
print "(--[ .]-[ .]"

if chins == 0:
    print "(_______O__)"
    sys.exit(0)
else:
    print "(       O  )"
    for i in range(1, chins):
        print "(          )"
    print "(__________)"
